### ISSUE TITLE
!see command and filtering on tags bug

### ISSUE THREAD

**Issue by inico** [unclebob/fitnesse#985]
When using the !see command in Fitnesse, these pages are always included when executing a suite. This also occurs when applying a tag filter which should filter these results

repro:
create a suite with some tags (suite 1)
create another completely suite with tests which use the !see command (see referring to tests from suite 2) (suite 2)
run test with =suite&runTestsMatchingAllTags=regressietest,OV&format=xml&includehtml (in suite 1)
see that tests from suite 2 are included in the run, where these are not expected

**Comment by amolenaar** [comment]
Yep. In https://github.com/unclebob/fitnesse/blob/master/src/fitnesse/testrunner/SuiteContentsFinder.java#L34, all pages are collected, taking into account the suite filter. In the next line cross referenced pages are collected, but the suite filter is not applied here.
